tests/Behat/Element/Admin/OrderCreateFormElement.php

```diff
@@ -80,13 +80,39 @@ public function specifyBillingAddress(AddressInterface $address): void
         );
     }
 
+    public function getAvailableShippingMethods(): array
+    {
+        $this->clickOnTabAndWait('Shipments & Payments');
+
+        $shipmentsCollection = $this->getDocument()->find('css', '#sylius_admin_order_creation_new_order_shipments');
```

Comment:
Maybe move `#sylius_admin_order_creation_new_order_shipments` to `getDefinedElements(): array` method?